### PR TITLE
improve bounties sections

### PR DESCRIPTION
**Pull request by bwhm**
Some improvements, and examples for the Council, for bounties.

### CODE REVIEW

File: roles/council-members/README.md [hunk 1]
```diff
@@ -453,74 +453,211 @@ The main way of dealing with Leads is through the [proposal system](#proposals).
...
 
-##### Closed
-For a Bounty that requires investing lots of time and/or other resources, it may be reasonable to guarantee one or more Appliers that gets Assigned some time to complete all, or some, of the work, without having someone come in and "snipe" the reward.
+#### Definitions of Terms
+- Community Repo - this [Github repo](https://github.com/Joystream/community-repo/issues)
+- Bounty Issue - meaning a github issue made by `bwhm` or `blrhc` of Jsgenesis in the Community Repo
+- Council Member (CM) - a (current) member of the Joystream Council
+- Council Secretary - a CM elected by the other CMs to be the "main" representative of the Council towards Jsgenesis
+- Bounty Manager (BM) - a Community Member (can be a CM as well) elected by the Council to be responsible for managing all Bounties for some period of time, or "just" this specific Bounty. Defaults to the Council Secretary if no other person is chosen
+- Forum - the on chain [Joystream](https://testnet.joystream.org/#/forum/) forum
```
Comment (blrhc) [inline roles/council-members/README.md:473]:
```suggestion
- Forum - the on-chain [Joystream](https://testnet.joystream.org/#/forum/) forum
```

File: roles/council-members/README.md [hunk 2]
```diff
@@ -453,74 +453,211 @@ The main way of dealing with Leads is through the [proposal system](#proposals).
...
+Although the idea of forcing the Council to communicate and co-operate their way through this was noble, it leads to an example of the tragedy of the commons.
 
-##### Other
-In addition to the varieties outlined, other formats can be defined and chosen if they are more appropriate for a specific Bounty.
+As managing the Bounties will be a significant part of their (potential) KPI rewards, they have some excess resources and incentives to hire the "right" people. They can either pay for this out of pocket, request funding for this through spending proposals, reserve some part of the Bounty reward for this purpose or by asking Jsgenesis for designated funding.
 
-A "new" Council must honor any agreements and rules set by their predecessors, for as long as the rules say so.
+In the end, the Council will still be responsible for the work the BM does, so they should still verify the spending and/or text proposals made, and seek feedback of the BMs performance (eg. responsiveness and communication) from participants.
```
Comment (blrhc) [inline roles/council-members/README.md:482]:
```suggestion
In the end, the Council will still be responsible for the work the Bounty Manager does, so they should still verify the spending and/or text proposals made, and seek feedback of the Bounty Manager's performance (eg. responsiveness and communication) from participants.
```

File: roles/council-members/README.md [hunk 3]
```diff
@@ -453,74 +453,211 @@ The main way of dealing with Leads is through the [proposal system](#proposals).
...
-The workflow will depend both on the [Reward Distribution](#reward-distribution) and the [Format](#format), and must be established beforehand.
+#### General Steps
+Jsgenesis creates a Bounty Issue in the Community Repo with a new Bounty. This Bounty Issue is meant for the Council first and foremost, and ideally, the person(s) attempting, or successfully manages, to solve the Bounty should not even *need* to read this.
 
-- For "Closed" formats, an Applier must present a bid why they should be assigned the given Bounty. This should include detailed terms, such as time needed, costs, etc. If approved, this makes the terms valid.
-- In some cases, it may make sense to break a Bounty up in to milestones, with partial rewards at each stage. This builds trust as the Council can see the progress being made, and the Assignee can get chunks of the reward along the way.
-- In other cases, the person may need some initial funding to get started.
-- For "Closed" formats, the specifics of the workflow could be part of the Applier's application for participation.
+In this Bounty Issue, Jsgenesis will explain/outline:
+- what the bounties is for
```
Comment (blrhc) [inline roles/council-members/README.md:488]:
```suggestion
- what the bounty is for
```

File: roles/council-members/README.md [hunk 4]
```diff
@@ -453,74 +453,211 @@ The main way of dealing with Leads is through the [proposal system](#proposals).
...
+- the Bounty Manager
+  - deploying the Bounty
+  - advising the Council
+  - initial grading
+- the Council
+  - approving the Bounty
+  - voting on spending proposals
+- Applicants
+  - Community Members applying to be Assigned the Bounty
+- Assigned
```
Comment (blrhc) [inline roles/council-members/README.md:615]:
```suggestion
- Assignee
```

**Review by blrhc — APPROVED** [review]
I wonder whether `Assigned` -> `Assignee`, but other than that looks good.
I think they're equally valid in this context perhaps.

**Comment by bwhm** [comment]
> I wonder whether `Assigned` -> `Assignee`, but other than that looks good.
> I think they're equally valid in this context perhaps.

Probably correct. Feel free to suggest changes to that, and other smaller things!

**Pull request merged**